lib/approval_request_listener.rb

```diff
@@ -43,9 +43,9 @@ def process_event(topic)
       Rails.logger.info("Starting update_and_log_state for Approval ID: #{approval.id} with payload: #{topic.payload}")
       update_and_log_state(approval, topic)
       Rails.logger.info("Finished update_and_log_state for Approval ID: #{approval.id} with payload: #{topic.payload}")
-      Rails.logger.info("Staring Catalog::OrderItemTransition for order_item_id: #{approval.order_item_id}")
-      Catalog::OrderItemTransition.new(approval.order_item_id).process
-      Rails.logger.info("Finished Catalog::OrderItemTransition for order_item_id: #{approval.order_item_id}")
+      Rails.logger.info("Staring Catalog::ApprovalTransition for order_item_id: #{approval.order_item_id}")
```

Comment:
Do we still need this much logger detail? Or possibly could they be moved to `.debug`?

Reply:
I think it makes sense to make it debug now